test/pummel/test-crypto-dh.js

```diff
@@ -8,15 +8,32 @@ if (!common.hasCrypto) {
   return;
 }
 
-assert.throws(function() {
-  crypto.getDiffieHellman('unknown-group');
-});
-assert.throws(function() {
-  crypto.getDiffieHellman('modp1').setPrivateKey('');
-});
-assert.throws(function() {
-  crypto.getDiffieHellman('modp1').setPublicKey('');
-});
+assert.throws(
+  function() {
+    crypto.getDiffieHellman('unknown-group');
+  },
+  /^Error: Unknown group$/,
+  'crypto.getDiffieHellman(\'unknown-group\') ' +
+  'failed to throw the expected error.'
+);
+assert.throws(
+  function() {
+    crypto.getDiffieHellman('modp1').setPrivateKey('');
+  },
+  new RegExp(/^TypeError: crypto\.getDiffieHellman\(\.\.\.\)\./.source +
```

Comment:
You can work with strings here. You don't need to pass regular expression literals to the `RegExp` constructor.